src/d3a/constants.py

```diff
@@ -35,3 +35,10 @@
 # Controls how often will event tick be dispatched to external connections. Defaults to
 # 20% of the slot length
 DISPATCH_EVENT_TICK_FREQUENCY_PERCENT = 20
+
+COLLABORATION_ID = ""
+# Controls whether the external connection is for use with the redis api client
+# or with th3 d3a-web.
+# False -> Redis api client
+# True -> d3a-web
+EXTERNAL_CONNECTION_REDIS_OR_WEB = False
```

Comment:
Minor but EXTERNAL_CONNECTION_WEB would fit better IMO, because it is a boolean variable.

Reply:
Agreed, fixing this